apps/cookbook/src/app/layout-recipes/grid-layout/grid-layout-single-container/grid-layout-single-container.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+declare var require: any;
```

Comment:
Have you considered instead to add "node" under compilerOptions types in [tsconfig.app.json](https://github.com/kirbydesign/designsystem/blob/2d8dc84753c9f3b5ae6f73baa034cdd51fe24e36/apps/cookbook/tsconfig.app.json) as we already have the node types installed in the project?

Reply:
No, I didn't consider that. This was copied from existing showcase components. It seems to be the way it has been done so far. There's approx. 25 occurrences of `declare var require: any;` in as many files.

I do feel tempted to change it as you've suggested, but I think it's a bit out of scope for this issue - at least changing it in 20+ unrelated files. 

Another option is to create a small housekeeping issue and resolve it immediately after this PR.

Reply:
I've created a new issue for this: https://github.com/kirbydesign/designsystem/issues/1716

Reply:
...and another PR: https://github.com/kirbydesign/designsystem/pull/1717